CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-rc.0
+
+<!-- generated comparing v5.0.0-beta.5..next -->
+
+_Sep 1, 2021_
+
+A big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Renamed packages to `@mui/*`. This is part of our effort to expand the scope of the library beyond Material Design. Most of the effort on v5 was about customizability. More details about the [plan](https://github.com/mui-org/material-ui/issues/27825) and the [why](https://github.com/mui-org/material-ui/discussions/27803)
+- 🛠 Added `mui-replace` codemod for migrating `@material-ui/*` to new packages `@mui/*`. Check out this [codemod detail](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-codemod/README.md#mui-replace) or head to [migration guide](https://next.material-ui.com/guides/migration-v4/#preset-safe)
+- 🧪 Added new `<Mansory>` component to the lab, [check it out](https://next.material-ui.com/components/masonry/). It has been crafted by our first intern, @hbjORbj 👏!
+
+### `@mui/material@5.0.0-rc.0`
+
+#### Breaking changes
+
+- &#8203;<!-- 33 -->[core] Rename packages (#28049) @mnajdova
+
+  The packages were renamed. Here is the mapping between the old and new names:
```

Comment:
I would highlight that `@mui/core` is not the same as `@material-ui/core`. Generally, we just renamed the org. Except for `/core` and `/unstyled`